internals/secrethub/signup.go

```diff
@@ -109,6 +113,18 @@ func (cmd *SignUpCommand) Run() error {
 					return err
 				}
 			}
+			if cmd.org == "" {
+				cmd.org, err = ui.AskAndValidate(cmd.io, "Your organization (leave empty to skip for now): ", 2, allowEmpty(api.ValidateOrgName))
```

Comment:
What about:
```
Would you like to create a workspace for your organization right away? 
(This can be done later by running secrethub org init) [Y/n]
The name of your organization workspace:
```

Reply:
Why?

We'd like to encourage users to create the organization right away. With the proposed change, users have to answer an extra question to get there.
Users that do not want to create an organization, now also require more input (`n` + `ENTER`).

Reply:
To prevent a support flood for reclaims of dangling orgs.

Reply:
Oke, in that case, I think we should rethink whether we want to encourage creating the org right away in the first place.

Note that the workspace name (not the secrets) can be reclaimed even when access to the credential is lost. The flow is not yet optimal, but one could request a password reset via the dashboard and connect a CLI via `account init`, the account then has access to all namespaces it had before. Access to directories and secrets cannot be reset without a credential, as they are encrypted with the credential.

Reply:
First off, we discussed this at length before: yes we definitely want to encourage creating an organization on signup. It sets the stage for collaboration, very important for customer success.  

Second, let's take a page from the Slack playbook here for the copy: https://www.useronboard.com/how-slack-onboards-new-users/?slide=36

Something like this could work: 

- `Choose a shared workspace name for your team (e.g. your company, leave empty to skip for now):`
- `Team workspace name, e.g. your company (leave empty to skip):`
- `Team workspace name, usually your company name (leave empty to skip):`

Reply:
I like this one: `Team workspace name, usually your company name (leave empty to skip):`

Following the advice supplied in the guide you provided, should we provide extra context? e.g.

`Team workspace name (this is where your shared repositories will live), usually your company name (leave empty to skip):`

Reply:
`Workspace name for your team (e.g. your company, leave empty to skip for now):`

Reply:
Hypothesis: I think the concept of `workspace` is fairly familiar in the minds of our new users.

So IMO it's fine to leave out the `this is where your shared repositories will live` part. One of the feedback points we god for the signup flow was that it was a lot of text (especially the lines about where the credential file is stored, which we moved to the docs I believe), so I'd advocate for brief descriptive questions whenever possible. It's already a long-ish one.

Reply:
Now that we've changed the prompt to say workspace and nothing about an organization, should we change the flag names and descriptions too, to refer to team workspaces rather than organizations?